sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -448,6 +449,42 @@ def is_buffered_correctly(actual):
 
       assert_that(actual, is_buffered_correctly)
 
+  def test_deduplication_transform_with_processing_time(self):
+    # Note that current FnApiRunner doesn't respect either real timestamp.
```

Comment:
IIRC, the FnApiRunner does respect timestamps. (Not sure about the Create transform though.)